src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -213,6 +213,8 @@ public function findByIds(array $ids): array
     public function findDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
     {
         $this->assertEntityType($entityType, ['Clip', 'Episode', 'Series', 'Franchise', 'Gallery', 'Collection', 'Season']);
+        $streamableEntities = ['Episode', 'Clip', 'Series'];
```

Comment:
Yeah, I asked to @BjornTwachtmann about it. Also I saw this in pages-services and I tested it:
https://github.com/bbc/programmes-pages-service/blob/master/src/Data/ProgrammesDb/Entity/Programme.php#L50